### PR TITLE
[sample] Make the led blink sos in morse

### PR DESCRIPTION
**Pull request by ardumont**

### CODE REVIEW

File: examples/blink-led-morse.ms [hunk 1]
```diff
@@ -0,0 +1,39 @@
+;; @ardumont
+;; microscheme.org
+
+(include "src/stdlib.ms")
```
Comment (ardumont) [inline examples/blink-led-morse.ms]: The `(include "libraries/list.ms") did not work for me here.

Comment (ardumont) [inline examples/blink-led-morse.ms]: Is this the right way to include the standard library?

Comment (ryansuchocki) [inline examples/blink-led-morse.ms]: Hi Antoine. The standard library is actually loaded automatically, so there's no need to include it. (The list.ms, vector.ms and io.ms libraries are now redundant.) Thanks for the contribution...

Comment (ardumont) [inline examples/blink-led-morse.ms]: Thanks!
Indeed, removing the include still works!
I'll fix!

**Comment by ardumont** [comment]
I removed the now useless `include`.

**Pull request merged**